ctapipe/irf/irf_classes.py

```diff
@@ -90,9 +92,10 @@ class IrfToolBase(Tool):
         help="Prefix of the classifier `_prediction` column",
     ).tag(config=True)
 
+class EventPreSelector(Component):
     preselect_criteria = List(
```

Comment:
This should simply use a QualityQuery component, rather than specifying it separately here. That way if we fix bugs or change the way QualityQuery works, this code will still work.

Reply:
In fact, you could just inherit from QualityQuery here and add the renaming options (or simply remove them as I don't think you really need to rename columns as long as you keep track of them nicely). Then there is no need for this component at all ,just replace it with a QualityQuery as we do elsewhere

Reply:
Do you have a particular example of inheriting from quality query in mind?

Reply:
yes, you can see in for example ImageProcessor (ImageQualityQuery) for example,  or HillasReconstructor (StereoQualityQuery)

Reply:
Changed to base on QualityQuery.